### DISCUSSION TITLE
Github Action

### DISCUSSION THREAD

**Discussion by EMiller34** [community/community#48912] [Actions] [answered]
### Select Topic Area

Question

### Body

Hello! I have a quick question.  I'm a pm looking to put a simple control on a project I'm helping with.  I'm looking to create a github action:
On deployment, open wiki page. 

Can someone help me with the github action syntax for this?

**Comment by Akash1134** [comment, marked as answer]
Hi @EMiller34 , Thanks for reaching out. Do these [Marketplace Actions](https://github.com/marketplace?type=actions&query=GitHub+Wiki+) help you with what you're looking for?

**Comment by Anmol-Baranwal** [comment]
You can learn about these from organizational repositories & can also use several bots like triage, which you can sure find at marketplace.

Some repos to check for github actions: (Reading docs is recommended)
- https://github.com/EddieHubCommunity/LinkFree/tree/main/.github
- https://github.com/TheAlgorithms/C-Plus-Plus/tree/master/.github
- https://github.com/ZeroOctave/ZeroOctave-Javascript-Projects/tree/main/.github/workflows

**Reply by Anmol-Baranwal** [reply]
So, if there are no further doubts. Could you mark any answer as answered and let others know that question has been answered.

**Reply by EMiller34** [reply]
Thank you! Looking through the marketplace and these repositories definitely helped!
